content/docs/sidebar.json

```
@@ -525,5 +525,33 @@
         "slug": "dvclive-with-dvc"
       }
     ]
+  },
+  {
+    "label": "Studio",
+    "slug": "studio",
+    "source": "studio/index.md",
+    "icon": "studio",
+    "children": [
+      {
+        "label": "Get started",
+        "slug": "get-started"
+      },
```

Slugs get turned into titles automatically 🙂 (`label` is there for when the slug doesn't match the title).

```suggestion
      "get-started",
```